### ISSUE TITLE
Gradients are always zero during training

### ISSUE THREAD

**Issue by nmerci** [tensorflow/tensorflow#3170]
I have implemented [YOLO](http://pjreddie.com/media/files/papers/yolo.pdf) algorithm in tensorflow and face with an issue during training: all my gradients are equal to zero from the very beginning of training. I also noticed that with some weights initialization gradients become non-zero at first several iterations and then stick to zero. Thanks in advance for any help!

### Environment info
Operating System: Ubuntu 16.04 lts
Tensorflow version: 0.8.0

**Comment by nmerci** [comment]
The issue has been fixed by itself.

**Comment by samrat1997** [comment]
Can you explain a bit about how you fixed it ?

**Comment by Mazecreator** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  mazecreator@gmail.com
    Domain mazecreator.com has exceeded the max emails per hour (28/25 (112%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext6.iad.github.net ([192.30.252.197]:51228 helo=github-smtp2b-ext-cp1-prd.iad.github.net)
	by server2.lowesthostingrates.com with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.89)
	(envelope-from <noreply@github.com>)
	id 1demwa-0005nS-Ff
	for mazecreator@mazecreator.com; Mon, 07 Aug 2017 13:40:30 -0500
Date: Mon, 07 Aug 2017 11:50:53 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1502131853;
	bh=wnmIVO8laFF0/QYZjd0HsbZDsIKi1MP3qQarBGXHvo4=;
	h=From:Reply-To:To:Cc:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=ojwXFYfc7BzmEQlBPlWzcORr6EGUAPGYzbPOJJNqWanb+B7ecIPDoxeGNxUvaW92x
	 tJ7vYUNoUXaVAPBO/5O8UGQv9JWh6Gveg8bxEkc9wXZFa+XW7twYtIFj23skNjBTuA
	 rIeZFIvwcnuRkgxYY+dT5/pA7bxCE8LExK99pLNk=
